integration_tests/test_cfngin/tests/16_aws_lambda_hook.py

```diff
@@ -1,61 +1,81 @@
-"""Test AWS Lambda hook."""

```

Comment:
This file is not nearly as changed as the git diff is implying here. The main change is the `_build` method.